tests/playwright/src/model/pages/resources-page.ts

```diff
@@ -26,6 +29,10 @@ export class ResourcesPage extends SettingsPage {
   readonly podmanResources: Locator;
   readonly composeResources: Locator;
   readonly kindResources: Locator;
+  readonly kindProviderSetup: Locator;
```

Comment:
I would say this does not belong here. Every "Provider connection card" on the resources page is generated dynamically. We should also approach it this way, thus, let's create an abstract/general class to cover shared functionality and behavior of the every resource connection card, and then we can add specific provider card  implementations if that differs. Something similar as in Extensions.